.github/workflows/nightly-next.yml

```diff
@@ -28,7 +28,9 @@ jobs:
           registry-url: https://registry.npmjs.org/
       - name: Setup and publish nightly
         run: |
-          npm ci
+          npm i
+          npm i zrender@npm:zrender-nightly@next
```

Comment:
Should we change the order to
```ts
# update echarts version
node build/prepareNightly.js --next
# update zrender nightly to latest
npm i zrender@npm:zrender-nightly@next
# install other dependencies.
npm ci
```

Seems it will be more reasonable

Reply:
Looks good.

Reply:
And should we also change the exported `version` to the nightly version to keep consistent with the `package.json`?

Reply:
It's good to have. But I think it's also fine to keep unchanged if it's too complex to fix